app/services/parse/bing_parse_service.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+module Parse
+  class BingParseService
+    SELECTORS = {
+      adwords_top: 'div.b_imagePair.square_xb div.inner a',
+      adwords_body: 'div.b_adurl a',
+      non_adwords: 'li.b_algo h2 a, div.pagereco_TTitle a',
+      href: 'href'
+    }.freeze
+
+    def initialize(html_code)
+      raise IcRailsHueyToby::Errors::ParseServiceError unless html_code
+
+      @html_code = html_code
+    end
+
+    def parse
+      doc = Nokogiri::HTML(html_code)
+
+      adword_list = parse_adword_list(doc)
+      non_adword_list = parse_non_adword_list(doc)
+      total_links_count = parse_total_links_count(adword_list, non_adword_list)
+
+      Hash.new.merge(adword_list, non_adword_list, total_links_count) # rubocop:disable Style/EmptyLiteral
```

Comment:
Can we change to `{}` instead of `Hash.new` to avoid [this warning](https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/EmptyLiteral)?
```suggestion
      {}.merge(adword_list, non_adword_list, total_links_count)
```